app-frontend/src/app/components/projects/projectItem/projectItem.controller.js

```diff
@@ -25,12 +28,13 @@ export default class ProjectItemController {
                 this.selectedStatus = selected;
             }
         );
-        this.getMap = () => this.mapService.getMap(`${this.project.id}-map`);
+        this.mapId = `${this.project.id}-map`;
+        this.getMap = () => this.mapService.getMap(this.mapId);
+
+        this.showProjectThumbnail =
+            this.featureFlags.isOnByDefault('project-preview-mini-map');
```

Comment:
So this doesn't seem quite right to me.

If the feature flag is `active` we want to show the mini-maps. This does the opposite. I think `this.showProjectThumbnail` should be true if the feature flag is false.